app/controllers/admin/routes.py

```diff
@@ -28,7 +30,10 @@
 from app.logic.fileHandler import FileHandler
 from app.logic.bonner import getBonnerCohorts, makeBonnerXls, rsvpForBonnerCohort
 from app.controllers.admin import admin_bp
-from app.logic.serviceLearningCoursesData import parseUploadedFile, courseParticipantPreviewSessionCleaner
+from app.logic.manageSLFaculty import getCourseDict
+from app.logic.courseManagement import unapprovedCourses, approvedCourses
+from app.logic.utils import getRedirectTarget, setRedirectTarget
```

Comment:
This is the second import from utils, these imports should probably be moved to the same line.